test/Streamly/Test/Prelude/Serial.hs

```diff
@@ -106,55 +107,192 @@ splitOnSuffixSeq = do
         S.toList
              $ IS.splitOnSuffixSeq (A.fromList pat) (FL.toList) (S.fromList xs)
 
+seqSplitterProperties ::
+       forall a. (Arbitrary a, Eq a, Show a, Storable a, Enum a)
+    => a
+    -> String
+    -> Spec
+seqSplitterProperties sep desc = do
+    describe (desc <> " splitOnSeq")
+        $ do
+            -- Empty seperator
+            intercalateSplitEqId 0
+            concatSplitIntercaEqConcat splitOnSeq_ intercalate IS.intercalate 0
```

Comment:
In all these functions - why are we passing `intercalate` and `IS.intercalate`? We can use them directly without passing.

Reply:
We need to pass `intercalateSuffix` in the case of `splitOnSuffixSeq_`